crates/matrix-sdk/Cargo.toml

```diff
@@ -96,6 +98,7 @@ matrix-sdk-sqlite = { workspace = true, optional = true }
 matrix-sdk-test = { workspace = true, optional = true }
 mime = "0.3.16"
 mime2ext = "0.1.52"
+openidconnect = { git = "https://github.com/poljar/openidconnect-rs", rev = "c7e1dc31b83dd7559125984bfd36b9c0f191585e" }
```

Comment:
Do you have a PR for your fork https://github.com/poljar/openidconnect-rs?

Do you have a PR for your fork https://github.com/poljar/oauth2-rs?

Reply:
No, not yet. The fork only adds the `Send` trait bound on one of the things in those crates. We don't need this ourselves, but `uniffi` requires futures to be `Send` for them to be exposed over the FFI. So only the bindings require this.